scheduler/reconcile_test.go

```diff
@@ -1271,33 +1286,42 @@ func TestReconciler_RescheduleLater_Batch(t *testing.T) {
 		createDeployment:  nil,
```

Comment:
Should update one of these tests to create a migrate stanza and verify that we don't create a deployment for reschedules.